morepath/directive.py

```diff
@@ -1094,16 +1076,22 @@ def __init__(self):
         identity policy is in use. So you can pass some settings directly to
         the IdentityPolicy class.
         """
-        pass
 
-    def actions(self, obj):
-        yield IdentityPolicyFunctionAction(generic.identify,
-                                           'identify'), obj
+    def identifier(self, identity_policy_registry, setting_registry):
+        return ()
+
+    def perform(self, obj, identity_policy_registry, setting_registry):
```

Comment:
Ah, here an identity policy object is created which then presumably is delegated from the app class. Thanks to ``app_class_arg`` this is not needed; we just directly assign the methods from identity policy to the app class here.